sdk/core/core-lro/docs/LROEngine.md

```diff
@@ -0,0 +1,214 @@
+# Modular Support for Long-Running Operations
+
+Long-running operations (LROs) are operations that the service _could_ take a long time to finish processing and they follow a common convention:
+
+- the customer first send a an initiation request to the service, which in turn sends back a response, from which the customer can learn how to poll for the status of the operation, if it has not been completed already,
```

Comment:
Please pass this text through Word or Grammarly to ensure it’s properly cased, and to ensure it’s free from typos

Reply:
Done.